### PR TITLE
Fix TL;DR section to mention wait after operator create so that user …

### PR DESCRIPTION
**Pull request by vasukulkarni**
Fix TL;DR section to mention wait after operator create so that user will not try copy/paste and
be unlucky.

Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

[skip ci]

### CODE REVIEW

File: Documentation/ceph-quickstart.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ If you're feeling lucky, a simple Rook cluster can be created with the following
 ```
 cd cluster/examples/kubernetes/ceph
 kubectl create -f operator.yaml
+  # verify the rook-ceph-operator, rook-ceph-agent, and rook-discover pods are in the `Running` state before proceeding
```
Comment (galexrt) [inline Documentation/ceph-quickstart.md]: Could you please remove the extra spaces in front of the lines you added

**Comment by vasukulkarni** [comment]
@travisn I did copy/paste both the command and I only saw the operator pods and not cluster, rerunning cluster just said "customresourcedefinitions.apiextensions.k8s.io "cephclusters.ceph.rook.io" already exists and nothing worked later on.

**Comment by travisn** [comment]
@vasukulkarni Agreed, there was some troubleshooting needed. If you are still having issues, let's discuss in slack what you're hitting. For this document, since the same statement about confirming the running pods is just a few lines later in the next section, I'd suggest we not add this to the tldr section.

**Comment by vasukulkarni** [comment]
Closing this, I will file an issue since you are saying this should work without waiting and should not have negative effects.

**Pull request closed without merging**